spec/caskinstaller_spec.rb

```diff
@@ -57,7 +70,7 @@ def do_install
 
     context "when cask is installed" do
       before do
-        allow(Bundle::CaskInstaller).to receive(:installed_casks).and_return(["google-chrome"])
+        allow(Bundle::CaskInstaller).to receive(:installed_casks).and_return(["google-chrome","firefox"])
```

Comment:
I don't really know. This comes from #94.
It does not seem relevant to me. I will remove this.